playwright/e2e/rosa-hosted/rosa-cluster-hosted-wizard-validation.spec.ts

```
@@ -218,23 +218,23 @@ test.describe.serial(
+        clusterFieldValidations.ClusterSettings.Machinepool.NodeCount.LowerLimitError,
       );
       await createRosaWizardPage.computeNodeCountIncrementButton().click();
       await createRosaWizardPage.isTextContainsInPage(
-        clusterFieldValidations.ClusterSettings.Machinepool.NodeCount.MultiZone.LowerLimitError,
+        clusterFieldValidations.ClusterSettings.Machinepool.NodeCount.LowerLimitError,
         false,
       );
       await createRosaWizardPage.selectComputeNodeCount(maxNodes);
       await expect(createRosaWizardPage.computeNodeCountIncrementButton()).not.toBeEnabled();
       await expect(createRosaWizardPage.computeNodeCountDecrementButton()).toBeEnabled();
       await createRosaWizardPage.selectComputeNodeCount((parseInt(maxNodes) + 1).toString());
```

ESLint

[error] Missing radix parameter.

(radix)